README.md

```diff
@@ -1,5 +1,5 @@
 <img src="https://raw.githubusercontent.com/ScratchAddons/ScratchAddons/master/images/icon.svg" alt="Scratch Addons logo" align="right" width="128px"></img>
-# Welcome to Scratch Addons' repository!
+<h1 align='center'>Welcome to the Scratch Addons repository!</h1>
```

Comment:
Hmm, HTML in markdown?

Reply:
Yup. Some limited tags are supported (That's also what Hans did to right align the image). I fixed some grammar and I also aligned the text center. I also added a paragraph at the top because I thought that just having the sheilds images and then the about header seemed really empty and disorganized.